### ISSUE TITLE
PurgeCss extractor doesn't support some ppx call format

### ISSUE THREAD

**Issue by gaku-sei** [dylanirlbeck/tailwind-ppx#115]
Unfortunately, ReasonMl will reformat the following:

```reason
switch (something) {
| `foo => [%tw "mt-1"]
| `bar => [%tw "mt-2"]
}
```

into

```reason
switch (something) {
| `foo =>
  %tw
  "mt-1"
| `bar =>
  %tw
  "mt-2"
}
```

(Which I find highly debatable, but that's an other story).

So the current extractor's regex (`/(?<=\[%tw ")[^\"]*(?=\"\])/g`) will not catch some classes.

(Notice that if there is a way to preserve the original ppx syntax application (`[%tw "..."]`), it's good for me ^^)

_edit_: I found the related issue in the Reasonml repo: https://github.com/facebook/reason/issues/2300

_edit 2_: This code too won't be caught by the current extractor:
```reason
[%tw
  "..."
]

**Comment by dylanirlbeck** [comment]
@gaku-sei Thanks for opening this issue -- I too do not agree with how `refmt` does the reformatting for that snippet. I'm thinking this issue wouldn't be terribly difficult to solve, we just need to extend the regex inside of our extractor to not care about whitespace or brackets (and instead focus on quotations). It might be more complex than that, but that's just an idea.

If you're interested in contributing, I think you would be a great person to put out a fix! The code for the extractor lives [here](https://github.com/dylanirlbeck/tailwind-ppx/blob/master/js/index.js), and the tests for the extractor live [here](https://github.com/dylanirlbeck/tailwind-ppx/blob/7d251b4b04703cead09de2d697e13d2e5b6cc9be/test/bucklescript/src/index_test.re#L28).

If you're busy, I'll try to get around to fixing this as soon as possible!

**Comment by dylanirlbeck** [comment]
@gaku-sei This issue should now be fixed in v0.7.9, which was just released on NPM. Let me know if you still encounter issues and I'll reopen the issue.
